src/components/ContributionsEpicReminder.tsx

```diff
@@ -0,0 +1,241 @@
+                <div data-target="close" className={closeButtonStyles} tabIndex={0}>
+                    <SvgClose />
+                </div>
+            </label>
+            <div className={paneStyles} data-target="pane">
+                <div className={lineWrapperStyles}>
+                    <Lines />
+                </div>
+                <div className={containerStyles}>
+                    <form data-target="form" data-reminder-date={reminderDate}>
+                        <h4 className={remindHeading}>Remind me in {reminderDateAsString}</h4>
+                        <div className={formWrapper}>
+                            <div className={inputWrapper}>
+                                <TextInput label="Email address" data-target="input" />
+                            </div>
+                            <Button
+                                iconSide="right"
+                                icon={<SvgArrowRightStraight />}
+                                data-target="submit"
+                                type="submit"
+                            >
+                                Set a reminder
+                            </Button>
+                        </div>
+                        <p className={errorTextStyles} data-target="error">
+                            We couldn&apos;t set a reminder for you this time. Please try again
+                            later.
+                        </p>
+                        <p className={formTextStyles}>
+                            We will use this to send you a single email in {reminderDateAsString}.
+                            To find out what personal data we collect and how we use it, please
+                            visit our{' '}
+                            <a
+                                className={linkStyles}
+                                href="https://www.theguardian.com/help/privacy-policy"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                Privacy Policy
+                            </a>
+                            .
+                        </p>
+                    </form>
+                    <div data-target="success">
+                        <h4 className={remindHeading}>Thank you! Your support is so valuable.</h4>
+                        <p className={successTextStyles}>
+                            We will be in touch to invite you to contribute. Look out for a message
+                            in your inbox in {reminderDateAsString}. If you have any questions about
+                            contributing, please{' '}
+                            <a
+                                href="mailto:contribution.support@theguardian.com"
+                                className={linkStyles}
+                            >
+                                contact us
+                            </a>
```

Comment:
Did you try using the [Link component](https://www.theguardian.design/2a1e5182b/p/43c26b-link/b/048fd1) for this?

Reply:
I did, and I'll need a custom theme to render the link with the colours I need. So I'm gonna keep as is - at the end of the day it's just an `<a>` tag with a single CSS property.

Reply:
I think `<Link priority="secondary">` should give you the `neutral[7]` text colour, unless I'm missing something else.